__tests__/app.test.js

```diff
@@ -111,3 +111,61 @@ describe('api/users', () => {
     })  
 })
 
+describe('PATCH api/articles/:article_id', () => {
+    it('should update the article vote property by given amount which is a positive number', () => {
+        const testInput = {inc_votes: 10}
+        return request(app)
+        .patch('/api/articles/1')
+        .send(testInput)
+        .expect(200)
+        .then((result) => {
+            expect(result.body.article.votes).toBe(110)
+            })
+        })
+    })
+    it('should update the article vote property by given amount which is a negative number', () => {
+        const testInput = {inc_votes: -10}
+        return request(app)
+        .patch('/api/articles/1')
```

Comment:
I would advise you test this -10 one on a different article_id compared to the previous test, it's just another way of checking it works on a different article, to give even more confidence that your code is doing what it should.